Cargo.toml

```diff
@@ -372,7 +372,8 @@ ocp-logging = [
   "sinks-prometheus",
   "sinks-gcp",
   "sinks-splunk_hec",
-  "sinks-http"
+  "sinks-http",
+  "sinks-socket",
```

Comment:
so, we are not introducing any new sink for syslog, we are using `socket` sink with syslog encoding.

Reply:
just a thought, perhaps a dedicated sink will give us better control over having parity with fluentd syslog output.

Reply:
Right, technically speaking I did not add a new sink, just a new encoding.